_posts/2022-02-23-sparse-models.md

```diff
@@ -54,107 +61,35 @@ LTH and successive works have shown that IMP and variants (such as Layer-adaptiv
-  </tr>
-  <tr>
-    <td class="tg-baqh">EfficientNet</td>
-    <td class="tg-baqh">3.59M</td>
-  </tr>
-  <tr>
-    <td class="tg-baqh">ResNet18</td>
-    <td class="tg-baqh">11.17M</td>
-  </tr>
-</tbody>
-</table>
-
-</div>
-
-The authors perform ticket search 50 times for each architecture on the CIFAR-10 dataset and compute
-trajectory lengths, percentage of surviving weights, test accuracy etc. after each stage of pruning.
-
-### Ticket Search Difficulty
-
-The success rate of finding winning tickets out of 50 trials is denoted as the `Ticket Search Difficulty` of a particular architecture.
-In the figures below, the architectures are ordered in increasing order of the number of parameters.
-
-<div align="center">
-{% include 2022-02-07-sparse-models/difficulty.html %}
-</div>
-
-For the architectures they tested, they conclude that, in general, it is `easier` to find winning tickets for smaller
-architectures; which is a contradictory to the assumption of LTH.
-
-### Quality of Winning Tickets
-
-Since larger architectures have more number of sub-networks, one would except to see the highest accuracy
-gain as compared to smaller architectures, as there is a higher chances of better-quality
-sub-networks to exist in larger architectures.
-
-
-<div align="center">
-{% include 2022-02-07-sparse-models/quality.html %}
-</div>
-
-Again, it is observed that ResNet18, which is the larget architecture has the lowest accuracy gain when
-compared to its dense counterpart.
+## Can we find Winning Tickets from Pre-trained models?
+
+One of the statements of LTH is that the network needs to be rewinded to its initial state. However, it is a common use-case that we take a large pre-trained model such as Wav2Vec2 and then finetune on a down-stream task which usually is a much smaller dataset.
+
+Can we treat these pre-trained weights as the "initial state" in LTH and be able to find sparse models? Authors from [3] show that this is true. They show that they were able to prune away upto 95% of weights while achieving comparable WER on multiple architectures, ranging from CNN-LSTM, RNN-Transducer to Conformer and two public datasets (Common Voice and LibriSpeech).
+
+<figure>
+<center>
+  <img alt="Ticket Search for Pretrained Models" src="/assets/images/posts/sparse-models/pretrained.jpeg"/>
+  <figcaption>Fig 2: Iterative Pruning for Pretrained Models. x-axis: Percentage of Weights remaining after each pruning stage. y-axis: Word Error Rate (WER) % </figcaption>
+</center>
+</figure>
+
+In Fig 2, $$\theta_0$$ denotes random initialization, $$\theta_{CV}$$ denotes transfering from weights trained on CommonVoice, and $$\theta_{Libri}$$ denotes transfering from weights trained on LibriSpeech. WER is reported on TED-LIUM dataset. 
+
+We can observe that, as expected, models initialized with pretrained weights have a lower WER, however, the drop in WER due to Ticket Search is lower but comparable to non-pretrained initialization.
 
 
 ## Conclusion
```

Comment:
To me, the post lacks clarity on its 'one overarching idea'. The title says that, after reading this, I will come out wiser in ways of speeding up inference using lottery ticket hypothesis.

But the post itself is generally about LTH and doesn't dive too deep, say via personal experimentation, to add on what's been already said.

Also the mention of robustness looks like something to add meat to the post and not helping in the core argument.

Reply:
This was because there's no clarity on who the audience is. If we want the blogs to sound more "researchy" then I can do that. If the blog is meant to show code examples, then that can be done too. I assumed the audience is for someone new to the field of Speech, yet understands the basics of ML. 

By depth, do you want me to go over papers that build on LTH and theoretically prove LTH and walk through the proofs?